src/test/java/systems/composable/dropwizard/cassandra/DropwizardCassandraIntegrationTest.java

```diff
@@ -36,22 +37,51 @@
 
     @ClassRule
     public static final DropwizardAppRule<SmokeTestConfiguration> APP =
-            new DropwizardAppRule<>(SmokeTestApp.class, Resources.getResource("minimal.yml").getPath());
+        new DropwizardAppRule<>(SmokeTestApp.class, Resources.getResource("minimal.yml").getPath());
 
-    @Test
-    public void canQueryCassandra() throws Exception {
-        final URI uri = UriBuilder.fromUri("http://localhost")
-                .port(APP.getLocalPort())
-                .path("query")
-                .build();
+    @ClassRule
+    public static final DropwizardAppRule<SmokeTestConfiguration> APP_INJ =
```

Comment:
I'm surprised the two app rules don't conflict with each other. Are you sure this is how they're supposed to be used? I had a brief dig into the source, but didn't manage to see how they do port allocation.

Ideally these should be separate test classes as they're testing different things. It's not immediately obvious that some test cases are checking injection.